insights/specs/insights_archive.py

```diff
@@ -39,6 +39,7 @@ class InsightsArchiveSpecs(Specs):
     docker_info = simple_file("insights_commands/docker_info")
     docker_list_containers = simple_file("insights_commands/docker_ps_--all_--no-trunc")
     docker_list_images = simple_file("insights_commands/docker_images_--all_--no-trunc_--digests")
+    docker_container_inspect = glob_file("insights_commands/docker_inspect_*")
```

Comment:
The file name is the same as the one in https://github.com/RedHatInsights/insights-core/pull/1195, it will puzzle the parser to parse both specs I think.

Reply:
Two workarounds I could think for this issue:
1. Add different options for each of them and make sure the option does not affect the actual output.
2. Update the parser and distinguish them inner the parser by analyzing their content.